eng/versioning/external_dependencies.txt

```diff
@@ -17,7 +17,6 @@ com.microsoft.azure:azure-client-runtime;1.7.3
 com.microsoft.azure:azure-core;0.9.8
 com.microsoft.azure:azure-keyvault-cryptography;1.2.2
 com.microsoft.azure:qpid-proton-j-extensions;1.2.3
-com.microsoft.azure:spotbugs-maven-plugin;1.2.1
```

Comment:
are we removing spotbugs here?

Reply:
This package doesn't exist and, because it doesn't exist, this dependency tag isn't used anywhere.

Reply:
Thank you @alzimmermsft!